source/libs/utils.js

```diff
@@ -1,14 +1,32 @@
 import {h} from 'dom-chef';
 import select from 'select-dom';
-import elementReady from 'element-ready';
 import domLoaded from 'dom-loaded';
+import elementReady from 'element-ready';
+import OptionsSync from 'webext-options-sync';
+
+const options = new OptionsSync().getAll();
 
 /**
+ * Enable toggling each feature via options.
  * Prevent fn's errors from blocking the remaining tasks.
  * https://github.com/sindresorhus/refined-github/issues/678
- * The code looks weird but it's synchronous and fn is called without args.
  */
-export const safely = async fn => fn();
+export const enableFeature = async (fn, fileName) => {
+	const {disabledFeatures} = await options;
+	const log = disabledFeatures.includes('log-active-features') ? () => {} : console.log;
```

Comment:
Why have this as an option instead of just enabled by default when in dev mode?

Reply:
Because it’s a user-facing feature also. One might want to see what’s enabled/disabled on a page. Maybe?

Reply:
Because it’s a user-facing feature also. One might want to see what’s enabled/disabled on a page. Maybe?

Reply:
Could we have a checkbox to enable dev/debug mode instead? It doesn’t really feel like a feature in that sense.

Reply:
What about enabling logging by default with `console.group()`, or even better `console.groupCollapsed()`, so that way console is not cluttered and the user still gets to see what features are enabled.

> It doesn’t really feel like a feature in that sense

👍

Reply:
I don't think extensions should clutter the console by default

Reply:
Let's go with a checkbox that is unchecked by default. It can be useful for other production debugging we might want later on too.